src/notebook/index.css

```diff
@@ -37,67 +37,87 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
```

Comment:
I'll change it to 7 for now and run it by @ellisonbg when he comes into the office later